src/modules/RootPart.rb

```diff
@@ -1201,9 +1201,36 @@ def CheckBootSize(bootpart)
       end
     end
 
+    # Mount /sys /proc and the like inside Installation.destdir
+    # @return [void]
+    def mount_specials_in_destdir
+      # mount sysfs first
+      if MountPartition("/sys", "sysfs", "sysfs") == nil
+        AddMountedPartition(
+          { :type => "mount", :device => "sysfs", :mntpt => "/sys" }
+        )
+      end
+
+      if MountPartition("/proc", "proc", "proc") == nil
+        AddMountedPartition(
+          { :type => "mount", :device => "proc", :mntpt => "/proc" }
+        )
+      end
+
+      # to have devices like /dev/cdrom and /dev/urandom in the chroot
+      if MountPartition("/dev", "devtmpfs", "devtmpfs") == nil
```

Comment:
OK, this also includes commit d5cf7a6c06303f14f85ba5994a48c80518647daa (which is in the OBS changelog but not in the git changelog)